lib/runner/TestRunner.js

```diff
@@ -0,0 +1,171 @@
+const chalk = require("chalk");
+const Configuration = require("./Configuration");
+const fs = require("fs");
+const InteractionResult = require("../test/TestResult").InteractionResult;
+const TestParser = require("../test/TestParser");
+const TestResult = require("../test/TestResult").TestResult;
+const Util = require("../util/Util");
+// eslint-disable-next-line no-unused-vars
+const VirtualAlexaInvoker = require("./VirtualAlexaInvoker");
```

Comment:
you can just require without storing in a variable for the methods to be present to avoid the disabling es-lint

Reply:
I do use the value - it is referenced here dynamically:
https://github.com/bespoken/skill-testing-ml/blob/MultiRunnerSupport/lib/runner/TestRunner.js#L46